samza-core/src/main/java/org/apache/samza/operators/StreamGraphSpec.java

```diff
@@ -206,19 +231,6 @@ public String getNextOpId(OpCode opCode) {
     return getNextOpId(opCode, null);
   }
 
-  /**
-   * See {@link StreamGraphSpec#getIntermediateStream(String, Serde, boolean)}.
-   *
-   * @param <M> type of messages in the intermediate stream
-   * @param streamId the id of the stream to be created
-   * @param serde the {@link Serde} to use for messages in the intermediate stream. If null, the default serde is used.
-   * @return  the intermediate {@link MessageStreamImpl}
-   */
-  @VisibleForTesting
-  public <M> IntermediateMessageStreamImpl<M> getIntermediateStream(String streamId, Serde<M> serde) {
-    return getIntermediateStream(streamId, serde, false);
-  }
-
   /**
    * Internal helper for {@link MessageStreamImpl} to add an intermediate {@link MessageStream} to the graph.
```

Comment:
Method is public now, so does the javadoc need to be updated?

Reply:
No, it's only public in the implementation, not in the StreamGraph interface. It's still meant as a framework internal method.